prov/verbs/src/fi_verbs.c

```diff
@@ -363,6 +363,33 @@ ssize_t fi_ibv_send_iov_flags(struct fi_ibv_msg_ep *ep, struct ibv_send_wr *wr,
 	return fi_ibv_send(ep, wr, len, count, context);
 }
 
+static void fi_ibv_get_param_int(char *param_name, char *param_str,
+				 size_t *param_default)
+{
+	char *param_help;
+	size_t len, ret;
+	int param;
+
+	len = strlen(param_str) + 50;
+	param_help = malloc(len);
+
+	ret = snprintf(param_help, len, "%s (default: %zu)", param_str,
+		       *param_default);
+	if (ret >= len) {
+		VERBS_WARN(FI_LOG_EP_DATA,
+			   "param_help string size insufficient!\n");
+		free(param_help);
+		assert(0);
```

Comment:
add a return here, so that a free build doesn't continue and crash (and to keep coverity from whining)